### ISSUE TITLE
POST method not working.

### ISSUE THREAD

**Issue by BrunoEleodoro** [Caesar2011/denotrain#5]
Hello Guys, nice work you all did !.

I have a question about the POST request:

here's my script
```
import { Application, Router } from "./deps.ts";

const app = new Application({ port: 3001 });

app.get("/", (ctx) => {
  return { "hello": "world" };
});

app.post("/cadastrar", async (ctx) => {
  console.log(ctx.req);
  return ctx.req.body;
});

await app.run();

```
<img width="509" alt="Screen Shot 2020-05-15 at 18 52 56" src="https://user-images.githubusercontent.com/20596317/82099263-4ce16480-96dd-11ea-9d83-78bf2366e223.png">

Postman

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/20596317/82099310-6da9ba00-96dd-11ea-8a65-d66818285a62.png">

**Comment by Caesar2011** [comment]
You are not setting a `Content-Type` header in your request headers. It has to be `application/json` to get parsed as a json. The code works as expected.

**Comment by BrunoEleodoro** [comment]
Please have a look on the output of the console.

I'm passing it as application/json

<img width="1110" alt="Screen Shot 2020-05-15 at 19 31 02" src="https://user-images.githubusercontent.com/20596317/82101581-3a6a2980-96e3-11ea-8c3a-de5557a67fe2.png">

**Comment by BrunoEleodoro** [comment]
Using CURL

<img width="743" alt="image" src="https://user-images.githubusercontent.com/20596317/82101628-5e2d6f80-96e3-11ea-8ab1-33bb2b61dbe8.png">

**Comment by BrunoEleodoro** [comment]
Nevermind, I've added this code and it started working.

Thanks.!

<img width="446" alt="Screen Shot 2020-05-15 at 20 00 33" src="https://user-images.githubusercontent.com/20596317/82102708-bd40b380-96e6-11ea-9ff9-2c3b32b30759.png">

**Comment by Caesar2011** [comment]
I am curious why a this stub cookie Middleware handler should make any difference to the body. I will investigate it further tomorrow.

**Comment by Caesar2011** [comment]
There is a racing condition. Will be fixed.

**Comment by BrunoEleodoro** [comment]
Appreciate your help on this, thanks @Caesar2011 !!

**Comment by Caesar2011** [comment]
Fixed in `v0.4.4`.
